perllib/FixMyStreet/Cobrand/Peterborough.pm

```diff
@@ -187,4 +187,103 @@ sub munge_report_new_contacts {
     $self->SUPER::munge_report_new_contacts($categories);
 }
 
+sub _premises_for_postcode {
+    my $self = shift;
+    my $pc = shift;
+
+    my $key = "peterborough:bartec:premises_for_postcode:$pc";
+
+    unless ( $self->{c}->session->{$key} ) {
+        my $bartec = $self->feature('bartec');
+        $bartec = Integrations::Bartec->new(%$bartec);
+        my $response = $bartec->Premises_Get($pc);
+
+        $self->{c}->session->{$key} = [ map { {
+            id => $_->{UPRN},
+            uprn => $_->{UPRN},
+            address => $self->_format_address($_),
+            latitude => $_->{Location}->{Metric}->{Latitude},
+            longitude => $_->{Location}->{Metric}->{Longitude},
+        } } @$response ];
+        # XXX Need to remove this from session at end of interaction
```

Comment:
Does this comment need actioning/removing? Or is it something for the future?

Reply:
It's removed again in eab6807abcfb235b89f3682d1ede795b009187dd :)